Sparkle/SUTouchBarButtonGroup.m

```diff
@@ -0,0 +1,60 @@
+//
+//  SUTouchBarButtonGroup.m
+//  Sparkle
+//
+//  Created by Yuxin Wang on 05/01/2017.
+//  Copyright © 2017 Sparkle Project. All rights reserved.
+//
+
+#import "SUTouchBarButtonGroup.h"
+
+@implementation SUTouchBarButtonGroup
+
+@synthesize buttons = _buttons;
+
+- (instancetype)initByReferencingButtons:(NSArray<NSButton *> *)buttons
+{
+    if (!(self = [super init]))
+        return self;
+
+    NSView *buttonGroup = [NSView new];
```

Comment:
+new is the same as [[NSView alloc] init]. In general, objc programmers don't use +new. An actual issue though is that -init is not documented for NSView so I think I'd be hesitant with invoking it. NSView's designated initializer is '-initWithFrame:' -- you can try using that,  even if it means supplying an empty rect, I suppose.